legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/ingestmode/digest/DigestGenerationHandler.java

```diff
@@ -106,6 +106,7 @@ else if (value instanceof StagedFilesFieldValue)
             .addAllValues(filteredStagingFieldValues)
             .addAllFieldTypes(filteredStagingFieldTypes)
             .putAllTypeConversionUdfNames(udfBasedDigestGenStrategy.typeConversionUdfNames())
+            .columnUdf(udfBasedDigestGenStrategy.columnTransformUdfName())
```

Comment:
let's call it: columnTransformationUdfName every where

Reply:
Sure changed!